sdk/core/core-lro/src/poller/models.ts

```diff
@@ -12,6 +12,10 @@ export interface OperationConfig {
   operationLocation?: string;
   /** The resource location */
   resourceLocation?: string;
+  /** The initial Url  */
+  initialUri?: string;
```

Comment:
lets call this `internalUrl` and change any other instances of `uri` to `url`

Reply:
I changed to `initialUrl` not `internalUrl` and the internal one is not a proper name for me. Because it may indicate this is an internal polling url.